src/style.js

```diff
@@ -35,6 +35,7 @@ export const dotColor = textLinkColor;
 export const selectedDotColor = foregroundColor;
 export const arrowColor = textLinkColor;
 export const monthTextColor = textDefaultColor;
+export const indicatorColor = '#808080'; // gray is the default of React Native ActivityIndicator
```

Comment:
Why not just pass undefined as default so the color will be set to the RN default color?

Reply:
Sounds great. I did not know, that this is working this way, when setting `undefined`.
How can I test that?
Thx, I will update my PR according to this later this week.

Reply:
Just run it with the undefined to see if the ActivityIndicator is running well.

Reply:
Please resolve the conflicts as well

Reply:
Sure.

Reply:
Yes, working.

![Bildschirmfoto 2019-05-08 um 13 08 44](https://user-images.githubusercontent.com/1942953/57371233-8611fb00-7192-11e9-8959-df1b6e513db3.png)